appinventor/blocklyeditor/src/workspace_svg.js

```diff
@@ -237,6 +237,7 @@ Blockly.WorkspaceSvg.prototype.addBackpack = function() {
         scrollbars: true,
         media: './assets/',
         disabledPatternId: this.options.disabledPatternId,
+        renderer: 'geras2_renderer',
```

Comment:
What was the issue here?

Reply:
The geras2_renderer provides the indented input rendering, so blocks with indented inputs (procedures or local scope blocks returning values) would render as regular right-aligned inputs.

Reply:
Ah, good catch!